### PR TITLE
Attempt to move cron jobs to k8s

### PR DESCRIPTION
**Pull request by mfcrocker**
# Motivation and Context
This moves responsibility for cron jobs in this service from Spring to K8s. The aim is to make it easier to understand which crons we have going on.

It also has some work to continue removing unused code from the service.

# What has changed
- Added a CronJob to the helm chart
- Added an endpoint back in to allow interaction from the CronJob
- Removed everything to do with Swagger
- Couple of bugfixes in the documentation

### CODE REVIEW

File: openapi.yaml [hunk 1]
```diff
@@ -69,228 +67,12 @@ paths:
...
-      parameters:
-        - name: actionType
-          in: path
-          description: the action type to map to the templates
-          required: true
-          schema:
-            type: string
+      summary: Triggers an export.
+      description: Triggers an export of actions to a print file.
       responses:
```
Comment (warrenbailey) [inline openapi.yaml:74]: I didn't realize these old endpoints were still in here 👍

File: src/main/java/uk/gov/ons/ctp/response/action/export/scheduled/ExportScheduler.java [hunk 2]
```diff
@@ -29,16 +28,10 @@ public ExportScheduler(
     this.appConfig = appConfig;
   }
 
-  /** Carry out scheduled actions according to configured cron expression */
-  @Scheduled(cron = "#{appConfig.exportSchedule.cronExpression}")
-  public void scheduleExport() {
+  // This is called using a K8s CronJob via /export
+  public void scheduleExport() throws Exception {
```
Comment (warrenbailey) [inline src/main/java/uk/gov/ons/ctp/response/action/export/scheduled/ExportScheduler.java:32]: This method could probably go then just make the processExport public and use that instead?

**Review by warrenbailey — APPROVED** [review]
Looks fine to me, a very minor comment about method names.

File: src/main/java/uk/gov/ons/ctp/response/action/export/scheduled/ExportScheduler.java (hunk 2 above)
Comment (mfcrocker) [reply src/main/java/uk/gov/ons/ctp/response/action/export/scheduled/ExportScheduler.java:32]: Yeah, I was just being lazy and didn't fancy changing the tests. Will get that done.

**Pull request merged**